src/common/snippets/include/snippets/lowered/pass/insert_broadcastmove.hpp

```diff
@@ -18,7 +18,7 @@ namespace pass {
  */
 class InsertBroadcastMove : public RangedPass {
 public:
-    OPENVINO_RTTI("InsertBroadcastMove", "RangedPass")
+    OPENVINO_RTTI("InsertBroadcastMove", "", RangedPass);
```

Comment:
Why `OPENVINO_MATCHER_PASS_RTTI` cannot be used for snippets passes?

Reply:
`OPENVINO_MATCHER_PASS_RTTI` takes up to 2 arguments and these classes don't derive from `MatcherPass`.